packages/react/src/components/index.ts

```diff
@@ -1,10 +1,11 @@
 export * from './asterisk/asterisk';
 export * from './button/button';
 export * from './checkbox/checkbox';
+export * from './field';
 export * from './field-label/field-label';
-export * from './field/field';
```

Comment:
How did you get this change?

I just checked locally, and to me it auto-formats as it was previously.

Reply:
I guess you still have the editor one enabled? Can you try to disable and see what result it does? Otherwise we will end up changing this with our each PR.

Reply:
Not ordering it is a Prettier offence

![image](https://user-images.githubusercontent.com/18623773/110650775-be315d00-81b2-11eb-9252-c8a8c091f9fc.png)

I can update our `.vscode/settings.json` with autoformat enabled, if you want, to make sure both our editors behave the same?

Reply:
So I guess what is actually needed also is for `.vscode/settings.json` to **disable** the autoformat of imports by the editor, right?

Reply:
Oh I'm so sorry, so stupid...

I just now noticed that the change was from `export * from './field/field';` to `export * from './field';` 🤦‍♂️ Obviously it would then change position!

All good, nothing to worry about...

Reply:
Yea it has also been reverted to `field/field` so should all be good.